### PR TITLE
Documentation for ista EcoTrend integration

### PR DESCRIPTION
**Pull request by tr4nt0r**
## Proposed change

Add documentation for ista EcoTrend integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118360
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5510
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ista_ecotrend.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- **Heating**: readings from your heat cost allocators (measured in units)
+- **Heating energy**: estimated value in kWh
+- **Heating costs**: estimated costs in EUR
+- **Hot water**: consumption readings in m³
+- **Hot water energy**: estimated value in kWh
+- **Hot water costs**: estimated costs in EUR
+- **Water**: consumption readings in m³
+- **Water costs**: estimated costs in EUR
+
+Not all values might be available in your ista EcoTrend account. Cost estimations is an optional service and has to be booked by your property manager. The cost sensors are therefore deactivated by default.
```
Comment (c0ffeeca7) [inline source/_integrations/ista_ecotrend.markdown]:
```suggestion
Not all values may be available in your ista EcoTrend account. Cost estimation is an optional service that has to be booked by your property manager. Therefore, the cost sensors are deactivated by default.
```

File: source/_integrations/ista_ecotrend.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Statistics
+
+The **ista EcoTrend** integration will additionally import all historical monthly consumption readings as statistics that can be used in the energy dashboard. New consumption readings usually become available in the first week of the following month. In contrast to the sensors, values in the statistics are corrected and added on the last day of their corresponding month. 
+
+Make sure to select the correct statistics in the energy dashboard. You can differentiate the statistics from the sensors by their prefix `ista_ecotrend`
```
Comment (c0ffeeca7) [inline source/_integrations/ista_ecotrend.markdown]:
```suggestion
Make sure to select the correct statistics in the energy dashboard. You can differentiate the statistics from the sensors by their prefix `ista_ecotrend`.
```

File: source/_integrations/ista_ecotrend.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ha_codeowners:
+  - '@tr4nt0r'
+ha_domain: ista_ecotrend
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **ista EcoTrend** integration for Home Assistant allows you to import your monthly meter readings from the [ista EcoTrend](https://ecotrend.ista.de) service.
```
Comment (c0ffeeca7) [inline source/_integrations/ista_ecotrend.markdown]:
```suggestion
The **ista EcoTrend** {% term integration %} for Home Assistant allows you to import your monthly meter readings from the [ista EcoTrend](https://ecotrend.ista.de) service.
```

File: source/_integrations/ista_ecotrend.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
+---
+title: ista EcoTrend
+description: Instructions on how to integrate ista EcoTrend with Home Assistant.
+ha_release: '2024.6'
```
Comment (c0ffeeca7) [inline source/_integrations/ista_ecotrend.markdown]:
```suggestion
ha_release: 2024.7
```

**Review by c0ffeeca7 — DISMISSED** [review]
Small tweaks

**Review by joostlek — APPROVED** [review]

**Pull request merged**